src/linker/Linker.Dataflow/JsonFlowAnnotationSource.cs

```diff
@@ -64,10 +64,18 @@ public DynamicallyAccessedMemberKinds GetThisParameterAnnotation (MethodDefiniti
 			return 0;
 		}
 
-		static DynamicallyAccessedMemberKinds ParseKinds(string s)
+		static DynamicallyAccessedMemberKinds ParseKinds(JsonElement attributes)
 		{
-			// Enum.Parse accepts a comma as a separator for Flags
-			return (DynamicallyAccessedMemberKinds)Enum.Parse (typeof (DynamicallyAccessedMemberKinds), s);
+			foreach (var attribute in attributes.EnumerateObject ()) {
+				if (attribute.Name == "System.Runtime.CompilerServices.DynamicallyAccessedMembersAttribute") {
```

Comment:
I think it would be nice to not require the `Attribute` suffix, just like the C# syntax doesn't require it,